t/app/controller/waste_peterborough.t

```diff
@@ -502,6 +476,55 @@ FixMyStreet::override_config {
     };
 };
 
+FixMyStreet::override_config {
+    ALLOWED_COBRANDS => 'peterborough',
+    MAPIT_URL => 'http://mapit.uk/',
+    COBRAND_FEATURES => { bartec => { peterborough => {
+        url => 'http://example.org/',
+        auth_url => 'http://auth.example.org/',
+        sample_data => 1 } },
+        waste => { peterborough => 1 },
+        waste_features => {
+            peterborough => {
+                max_requests_per_day   => 3,
+                max_properties_per_day => 1,
+            },
+        },
+    },
+    STAGING_FLAGS => {
+        send_reports => 1,
+    },
+}, sub {
+    my ($b) = shared_bartec_mocks();
+
+    subtest 'test waste max-per-day' => sub {
+        SKIP: {
+            # MEMCACHED_HOST needs to be set to 'memcached.svc' in
+            # general.yml-example (or whatever config file you are using)
+            skip( "No memcached", 7 )
+                unless Memcached::set( 'waste-prop-test', 1 );
+            Memcached::delete("waste-prop-test");
+            Memcached::delete("waste-req-test");
+            $mech->get_ok('/waste/PE1 3NA:100090215480');
+            $mech->get_ok('/waste/PE1 3NA:100090215480');
+            $mech->get('/waste/PE1 3NA:100090215481');
```

Comment:
If we make this 489 rather than 481, 489 is in the Bartec.pm sample_data and so we won't get some uninitialized warnings when running the tests.